src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/EnumConverterTests.cs

```diff
@@ -46,10 +46,30 @@ public void ConvertDayOfWeek()
             json = JsonSerializer.Serialize((DayOfWeek)(-1), options);
             Assert.Equal(@"-1", json);
 
-            // Not permitting integers should throw
             options = new JsonSerializerOptions();
             options.Converters.Add(new JsonStringEnumConverter(allowIntegerValues: false));
+            // Not permitting integers should throw
```

Comment:
nit: I think the comment was okay in the previous location.

Reply:
Fixed.